Resources/doc/resource_owners/battlenet.md

```diff
@@ -0,0 +1,26 @@
+Step 2x: Setup Battlenet
+=====================
+First you will have to register your application on [Battlenet](https://dev.battle.net/). Check out the
+documentation for more information https://dev.battle.net/io-docs.
+
+Next configure a resource owner of type `battlenet` with appropriate
+`client_id`, `client_secret` and `scope`. Refer to the [Battlenet](https://dev.battle.net/io-docs) API documentation
```

Comment:
Should refer to `https://dev.battle.net/docs/oauth` instead of `https://dev.battle.net/io-docs`